assets/constants.ts

```diff
@@ -50,6 +50,8 @@ export default {
 
   "ACTION_ACCEPT_SUGGESTION": "accept",
   "ACTION_REJECT_SUGGESTION": "reject",
+  "STATUS_SUGGESTION_ACCEPTED": "accepted",
```

Comment:
We must have this flag defined somewhere in the codebase because the same string is saved in the datastore, can you please check and avoid creating duplicate constants?

Reply:
I have checked for the `'accepted'` constants and did not find any matching results. Hence I created these two constants.